src/ui/win32/DialogBase.cpp

```diff
@@ -93,71 +94,78 @@ _Use_decl_annotations_ void DialogBase::CreateModalWindow(const TCHAR* restrict
 _Use_decl_annotations_
 INT_PTR CALLBACK DialogBase::DialogProc(HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
 {
+    // Message handlers made to reduce explicit casting only
+    const auto _OnCommand = [this]([[maybe_unused]] HWND /*hwnd*/, int id, HWND hwndCtl, UINT codeNotify)
+    {
+        switch (codeNotify)
+        {
+            case EN_KILLFOCUS:
+            {
+                auto* pControlBinding = FindControlBinding(hwndCtl);
```

Comment:
Gave it an intermediate handler to not have to cast ourselves, the member function is unchanged.